python/lsst/sconsUtils/builders.py

```diff
@@ -155,7 +155,7 @@ def filesToTag(root=None, fileRegex=None, ignoreDirs=None):
         #
         # Remove files generated by swig
         #
-        for swigFile in [f for f in filenames if re.search(r"\.i$", f)]:
+        for swigFile in [f for f in filenames if f.endswith(".i")]:
```

Comment:
I'm sorry to see we still need this, but presumably removing SWIG support should be done on a different ticket.